core/text/formatters.py

```diff
@@ -421,6 +375,25 @@ def formatter_immune(m) -> ImmuneString:
     return ImmuneString(str(value))
 
 
+def get_all_phrase_and_prose_formatters() -> tuple[dict[str, str], dict[str, str]]:
+    """Returns dictionary of all phrase formatters and a dictionary of all prose formatters"""
```

Comment:
This is the only place you call this a "phrase formatter". I get it (not a word formatter) but it's a bit confusing. Elsewhere (e.g. in the lists above) we just call this a "formatter".

Think we should have some documentation of all the different formatter types and how/where they can be used since it's become even more complicated.

Reply:
I adjusted the names and description of the function. I'd be inclined to separate further documentation into a separate issue.